src/logging.c

```diff
@@ -17,11 +20,49 @@ void fprint_int(int fd, int num) {
   sys_write(fd, int_str, len);
 }
 
+// For troubleshooting purposes: output contents of /proc/self/cmdline to the
+// log file
+void _output_cmdline() {
+  char buf[1024];
+  int cmdline_fd = sys_open("/proc/self/cmdline", O_RDONLY, 0);
+  while (1) {
+    int bytes = sys_read(cmdline_fd, buf, sizeof(buf));
+    if (bytes <= 0) {
+      break;
+    }
+    log_info("cmdline: ");
+    if (log_level >= INFO) {
+      sys_write(audit_log_fd, buf, bytes);
+    }
+    log_info("\n");
+  }
```

Comment:
```suggestion
  }
  sys_close(cmdline_fd);
```

oops, we need to close the FD!